code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -124,7 +124,7 @@
 				to humans. As a human, silicons are required to both protect and obey you.",
 		))
 
-	if(CONFIG_GET(flag/enforce_human_authority))
+	if(CONFIG_GET(number/human_authority) > 1)
```

Comment:
check this against defines (the numbers like, have meanings right?)

Reply:
Switched to checking these more explicitly with defines now, hopefully the meaning is more clear